From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@AndyHerb__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@MikeRayMSFT - Could you please review this update from @AndyHerb pls?

tysonn commented:
@MikeRayMSFT - second notice on review of public PR

MikeRayMSFT commented:
Thanks Tyson. I’m trying to remember how to merge the changes.

From: Tyson Nevil [mailto:notifications@github.com]
Sent: Friday, April 1, 2016 11:33 AM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Mike Ray (SR CONTENT DEVELOPER SQL) <Mike.Ray@microsoft.com>
Subject: Re: [Azure/azure-content] Corrected the description for AKV Principal Secret (#6107)


@MikeRayMSFT<https://github.com/MikeRayMSFT> - second notice on review of public PR

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/6107#issuecomment-204504922>

tysonn commented:
I'll take that as sign off. Merging.

MikeRayMSFT commented:
Thank you and please merge

Commit:
fixed broken links

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into blob-storage-soft-delete-release

Commit:
fixed image proportions

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into blob-storage-soft-delete-release

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into blob-storage-soft-delete-release

Commit:
Fixed bug in sample .Net code

Commit:
edits for public preview

Commit:
change of phrase ofr feature description

Commit:
misc

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5291

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5299

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

Commit:
Added (some) portal screenshots

Commit:
Added border to portal screenshots

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

v-viksur committed:
(ForumSTeam) Requesting to update the hyperlink.

In this documentation under the section "Upgrade Backup Server to v2" "Update the protection agents" hyperlink is pointing to the top this documentation. It should point to "Update the Data Protection Manager protection agent" section. Please review and provide your feedback.

ktoliver committed:
edit pass: two Azure Log Integration articles

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

Ja-Dunn committed:
fix

Ja-Dunn committed:
fix

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

ktoliver committed:
more edits

ktoliver committed:
edit pass: Two Azure Log Integration articles

ktoliver committed:
Update security-azure-log-integration-get-started.md

ktoliver committed:
Update security-azure-log-integration-get-started.md

ktoliver committed:
Update security-azure-log-integration-get-started.md

CarlRabeler committed:
[sql db] column store in s3 and above

Commit:
Added all portal screenshots

Commit:
resize images

JYOTHIRMAISURI committed:
update to ITSM Connections article

Commit:
incorporated Vamshi's suggestions

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into blob-storage-soft-delete-release

Commit:
pull from upstream and fix broken link

Ja-Dunn committed:
edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

AjayKumar-N committed:
(ForumsTeam) The "Next steps" section in the document  "Share snapshot FAQ" needs to hyperlink correctly.

In the referenced document https://docs.microsoft.com/en-us/azure/storage/files/storage-snapshots-files#next-steps
 The "Next steps" section in the document  "Share snapshot FAQ" needs to hyperlink correctly to the correct section of the destined document. Right now, it points to the top of the document.

AshokKumar-P committed:
[ForumsTeam] Minor Typo

Minor Typo correction

AshokKumar-P committed:
[ForumsTeam] Minor Typo Correction

Minor Typo Correction

v-nagta committed:
Missing Icons Update

v-nagta committed:
update

kevinvngo committed:
Update for Content Freshness

No changes in this doc

Commit:
Update azure-data-lake-analytics-limits.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

nitinme committed:
first wip checkin

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

Ja-Dunn committed:
fix

Ja-Dunn committed:
fix

Ja-Dunn committed:
fix

rachel-msft committed:
remove extra hyphen

KumudD committed:
draft complete

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into quickstartSLBCLI

ktoliver committed:
Merge branch 'master' into 1203165

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

hpsin committed:
PKCE v1

hpsin committed:
fix code_verifier meaning

nitinme committed:
some more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

tamram committed:
adding new soft delete article to TOC

tamram committed:
adding new soft delete article to TOC

jozender committed:
Add files via upload

jozender committed:
Add files via upload

jozender committed:
Add files via upload

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

yanancai committed:
modify Open steps

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

nitinme committed:
adding links in TOC and INDEX

v-nagta committed:
updated

v-nagta committed:
updated

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-arm-quickstart

nitinme committed:
fixing the image

charwen committed:
Updated CenturyLink

charwen committed:
updated centurylink

hpsin committed:
Update ms.date

jozender committed:
Update virtual-machines-common-planned-maintenance.md

alekseys committed:
Update mongodb-feature-support.md

CarlRabeler committed:
Merge branch 'master' into cs1

tfitzmac committed:
updating event grid portal

jozender committed:
Update virtual-machines-common-planned-maintenance.md

Ja-Dunn committed:
fix

Ja-Dunn committed:
fixes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
adding an article for all connectors

KumudD committed:
added files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into newslbportal0320

KumudD committed:
fixed broken link

tfitzmac committed:
updating portal

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into quickstartSLBCLI

neilpeterson committed:
[AKS] added preview statement

KumudD committed:
tweak

neilpeterson committed:
updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

j-martens committed:
AML: Metadata updates

nitinme committed:
Updating the TOC file

nitinme committed:
minor acrolinx fix

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

Commit:
Update for item

charwen committed:
added "http://"

Commit:
change

neilpeterson committed:
updates

rachel-msft committed:
Update tutorial-design-database-using-azure-cli.md

Commit:
Missed deletion and redirect.

PRMerger10 committed:
Merge pull request #35802 from alekseys/patch-40

Update mongodb-feature-support.md

Commit:
Fixed next step link

PRMerger12 committed:
Merge pull request #35846 from rthorn17/management-groups

Update for item

v-geberr committed:
403 clarification

v-geberr committed:
edits

neilpeterson committed:
updates

v-geberr committed:
acro check

v-geberr committed:
aka link

tfitzmac committed:
revised

j-martens committed:
AML: author name fix

PRMerger17 committed:
Merge pull request #35672 from rachel-msft/smalledit

remove extra hyphen

PRMerger18 committed:
Merge pull request #35839 from neilpeterson/aks-add-preview

[AKS] added preview statement

gauravtiwarimicrosoft committed:
Fix NotBefore date-time format

Windows

Commit:
Merge pull request #35770 from charwen/patch-45

Updated CenturyLink

gauravtiwarimicrosoft committed:
Update scheduled-events.md

jozender committed:
added Next steps section again

scoped it to maintenance requiring a reboot

ArLucaID committed:
Update msi-how-to-use-vm-msi-token.md

jozender committed:
Update maintenance-and-updates.md

GitHubber17 committed:
Merge pull request #35843 from j-martens/master

AML: Metadata updates

tfitzmac committed:
removed images

j-martens committed:
AML: metadata updates

Lisaco88 committed:
Merge pull request #35844 from nitinme/nitinme-databricks-updates

Adding connector article

v-geberr committed:
moved up

GitHubber17 committed:
Merge pull request #35854 from v-geberr/0320-roadmap-fix

aka link

PRMerger12 committed:
Merge pull request #35857 from arluca/patch-6

Update msi-how-to-use-vm-msi-token.md

rwike77 committed:
removed containers DNS issue, updated open source info

PRMerger14 committed:
Merge pull request #35853 from v-geberr/0320-403-fix

0320 403 fix

tfitzmac committed:
changed url

CarlRabeler committed:
Merge branch 'master' into cs1

Lisaco88 committed:
Merge pull request #35808 from tfitzmac/0316portaleg

updating event grid portal

MichaelHauss committed:
Merge pull request #1 from tamram/tamram-soft-delete

 adding new soft delete article to TOC

PRMerger19 committed:
Merge pull request #35859 from rwike77/faq

removed containers DNS issue, updated open source info

PRMerger19 committed:
Merge pull request #35856 from gauravtiwarimicrosoft/patch-4

[Linux ]Fix NotBefore date-time format

PRMerger19 committed:
Merge pull request #35221 from CarlRabeler/cs1

[sql db] column store in s3 and above

PRMerger20 committed:
Merge pull request #35855 from gauravtiwarimicrosoft/patch-1

[Windows]Fix NotBefore date-time format

lenadroid committed:
Update for Azure Databricks + Event Hubs tutorials

tamram committed:
Update storage-redundancy.md

KumudD committed:
added new topic

PRMerger16 committed:
Merge pull request #35858 from j-martens/master

AML: metadata updates

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into slbdiag

j-martens committed:
AML: metadata updates

PRMerger20 committed:
Merge pull request #35864 from j-martens/master

AML: metadata updates

j-martens committed:
AML: Metadata updates

Commit:
Merge pull request #35862 from MicrosoftDocs/tamram-patch-3

Update storage-redundancy.md

PRMerger12 committed:
Merge pull request #35866 from j-martens/master

AML: Metadata updates

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

j-martens committed:
AML: Metadata update

lenadroid committed:
date + editor

MikeDodaro committed:
Fix typo in include file

PRMerger16 committed:
Merge pull request #35870 from j-martens/master

AML: Metadata update

j-martens committed:
AML: Metadata updates

MikeDodaro committed:
Fix grammer/syntax issues

Lisaco88 committed:
Merge pull request #35742 from jozender/master

adding Live Migration information to planned maintenance article

KumudD committed:
formatted images

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into slbdiag

PRMerger16 committed:
Merge pull request #35748 from yanancai/master

Change ADL failed vertex debug steps based on new design

PRMerger17 committed:
Merge pull request #35872 from j-martens/master

AML: Metadata update

j-martens committed:
AML: metadata updates

KumudD committed:
updated TOC

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into slbdiag

Lisaco88 committed:
Merge pull request #35871 from MikeDodaro/fixTypo

Fix typo in include file

Lisaco88 committed:
Merge pull request #35852 from jeffgilb/320.1

Missed deletion and redirect.

neilpeterson committed:
[ACS] added secrets

rwike77 committed:
fixed typo in script example

PRMerger18 committed:
Merge pull request #35878 from rwike77/clusterfix

fixed typo in script example

christiankuhtz committed:
added port release mechanism

zhiweiwangmsft committed:
New preview feature risky IP

New preview feature risky IP

zhiweiwangmsft committed:
revert format

PRMerger19 committed:
Merge pull request #35881 from christiankuhtz/patch-88

added port release mechanism

Commit:
Merge pull request #35882 from zhiweiwangmsft/patch-22

New preview feature risky IP

KumudD committed:
udated TOC

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into quickstartSLBCLI

KumudD committed:
TOC update

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into newslbportal0320

v-nagta committed:
updated

KumudD committed:
fixed formatting

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into slbdiag

Commit:
Merge pull request #35448 from AshokKumar-P/patch-25

[ForumsTeam] Minor Typo Correction

PRMerger10 committed:
Merge pull request #35444 from AjayKumar-N/patch-23

(ForumsTeam) The "Next steps" section in the document  "Share snapsho…

Commit:
Merge pull request #35724 from hpsin/PKCE-v1-endpoint

v1 endpoint also supports PKCE

PRMerger19 committed:
Merge pull request #34697 from MohitGargMSFT/patch-6

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5299

PRMerger13 committed:
Merge pull request #34593 from MohitGargMSFT/patch-5

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5291

PRMerger15 committed:
Merge pull request #35604 from kevinvngo/patch-20

Update for Content Freshness

PRMerger20 committed:
Merge pull request #35303 from JYOTHIRMAISURI/master

Update to ITSM Connections article

Commit:
Merge pull request #35049 from v-viksur/patch-25

(ForumSTeam) Requesting to update the hyperlink.

PRMerger15 committed:
Merge pull request #35447 from AshokKumar-P/patch-24

[ForumsTeam] Minor Typo

Commit:
Fixed cloud shell link.

Commit:
Fixed cloud shell link.

Commit:
Fixed cloud shell link.

Commit:
Fixed cloud shell link.

PRMerger18 committed:
Merge pull request #35888 from jimdial/patch-102

Fixed cloud shell link.

PRMerger18 committed:
Merge pull request #35889 from jimdial/patch-103

Fixed cloud shell link.

PRMerger18 committed:
Merge pull request #35890 from jimdial/patch-104

Fixed cloud shell link.

PRMerger18 committed:
Merge pull request #35891 from jimdial/patch-105

Fixed cloud shell link.

rayne-wiselman committed:
putting articles in right place, fixing metadata

christiankuhtz committed:
added port release mechanism

christiankuhtz committed:
date update

rayne-wiselman committed:
Update recovery-plan-overview.md

rayne-wiselman committed:
Update best-practices-availability-paired-regions.md

Commit:
Merge pull request #35893 from christiankuhtz/patch-89

added port release mechanism

rayne-wiselman committed:
Update recovery-plan-overview.md

rayne-wiselman committed:
Update recovery-plan-overview.md

rayne-wiselman committed:
Update site-recovery-create-recovery-plans.md

Commit:
automation : hrw doc updates

Lisaco88 committed:
Merge pull request #35612 from saveenr/patch-25

Update azure-data-lake-analytics-limits.md

Commit:
updates

ktoliver committed:
Merge pull request #35686 from KumudD/quickstartSLBCLI

Create a Standard Load Balancer using Azure CLI 2.0

ktoliver committed:
Merge pull request #35836 from KumudD/newslbportal0320

Create a Standard Load Balancer -portal

Lisaco88 committed:
Merge pull request #35874 from j-martens/master

AML: metadata updates

davidbritch committed:
Corrected include file link.

Linked to correct include file for all the pre-req's for using APNS.

rjagiewich committed:
Merge pull request #35863 from KumudD/slbdiag

Standard Load Balancer Diagnostics

ShawnJackson committed:
Merge pull request #35090 from Ja-Dunn/virtual-networks-name-resolution-for-vms-and-role-instances

edit pass: virtual-networks-name-resolution-for-vms-and-role-instances

ktoliver committed:
Merge pull request #35861 from lenadroid/master

Update for Azure Databricks + Event Hubs tutorials

Commit:
updates

itechedit committed:
Merge pull request #35877 from neilpeterson/acs-oms-update

[ACS] added secrets

ktoliver committed:
change Windows Explorer to File Explorer

change Windows Explorer to File Explorer

ktoliver committed:
delete space

delete space

Lisaco88 committed:
Merge pull request #35623 from nitinme/nitinme-databricks-arm-quickstart

Databricks ARM quickstart

sethmanheim committed:
Fixing cust issue #5999

PRMerger14 committed:
Merge pull request #35898 from davidbritch/patch-16

Corrected include file link.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #35895 from georgewallace/autohrwupdate

automation : hrw doc updates

PRMerger16 committed:
Merge pull request #35903 from sethmanheim/cust-fix

Fixing cust issue #5999

mimig1 committed:
Cosmos DB: Small correction to Conn Service topic.

ShawnJackson committed:
Merge pull request #35175 from ktoliver/1203165

edit pass: two Azure Log Integration articles

itechedit committed:
Update recovery-plan-overview.md

tfitzmac committed:
style changes

itechedit committed:
Update site-recovery-create-recovery-plans.md

v-alje committed:
Merge pull request #35905 from MicrosoftDocs/FromPublicRepo

From public repo

cherylmc committed:
Update vpn-gateway-howto-vnet-vnet-resource-manager-portal.md

KumudD committed:
fixed typo

christiankuhtz committed:
add scale out

itechedit committed:
Merge pull request #35892 from rayne-wiselman/rayne-march21

putting articles in right place, fixing metadata

christiankuhtz committed:
tweaks

christiankuhtz committed:
tweaks

PRMerger16 committed:
Merge pull request #35909 from mimig1/patch-227

Cosmos DB: Small correction to Conn Service topic.

PRMerger17 committed:
Merge pull request #35915 from KumudD/slbdiagupdate

fixed typo

MichaelHauss committed:
incorporate suggestions from itechedit and tamram

PRMerger17 committed:
Merge pull request #35913 from cherylmc/patch-1

Update vpn-gateway-howto-vnet-vnet-resource-manager-portal.md

PRMerger19 committed:
Merge pull request #35914 from christiankuhtz/patch-92

add scale out

Commit:
Updated per customer feedback

itechedit committed:
Merge pull request #30392 from MichaelHauss/blob-storage-soft-delete-release

Blob storage soft delete release

v-shils committed:
Merge pull request #35860 from tfitzmac/0320url

changed url

GitHubber17 committed:
Merge pull request #35504 from v-nagta/icons

Missing Icons Update

PRMerger20 committed:
Merge pull request #35923 from jeffgilb/321

Updated per customer feedback
